### PR TITLE
<feature> add raw capacity card

### PR DESCRIPTION
**Pull request by shdn-ibm**

### CODE REVIEW

**Review by afreen23 — DISMISSED** [review]

**Comment by bipuladh** [comment]
/hold
Need clarification around metrics.

File: src/components/raw-capacity-card/raw-capacity-card.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+import './raw-capacity-card.scss';
+
+const colorScale = ['#0166cc', '#d6d6d6'];
+
+const RawCapacityCard: React.FC<any> = (props)  => {
+  const { t } = useTranslation();
+  const [data, loaded, loadError] = useK8sWatchResource<StorageInstanceKind>(GetFlashSystemResource(props?.match?.params?.name, props?.match?.params?.namespace));
+  const name= loaded && !loadError? data?.[0]?.metadata.name: undefined;
+  
+  const [totalCapacity, ,] = usePrometheusQuery(
```
Comment (rexagod) [inline src/components/raw-capacity-card/raw-capacity-card.tsx:44]: Nit: You can lose the commas here.

File: src/components/raw-capacity-card/raw-capacity-card.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>{t('Physical Capacity Overview')}</DashboardCardTitle>
+      </DashboardCardHeader>
+      <DashboardCardBody className="flashsystem-raw-usage__container">
+        {loaded && !loadError && (
+          <>
+            <div className="flashsystem-raw-usage__item flashsystem-raw-usage__legend">
+              <ChartLegend
+                fill={colorScale[0]}
```
Comment (GowthamShanmugam) [inline src/components/raw-capacity-card/raw-capacity-card.tsx:76]: I feel we need to change color when used capacity is crossing a certain threshold for warning and danger. We are using #f0ab00 for warning and #c9190b for danger. @afreen23 thoughts?

File: src/components/raw-capacity-card/raw-capacity-card.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+import {GetFlashSystemResource} from '../../constants/resources';
+import {parseMetricData} from '../../selectors/promethues-utils';
+import { StorageInstanceKind } from '../../types';
+import './raw-capacity-card.scss';
+
+const colorScale = ['#0166cc', '#d6d6d6'];
+
+const RawCapacityCard: React.FC<any> = (props)  => {
+  const { t } = useTranslation();
+  const [data, loaded, loadError] = useK8sWatchResource<StorageInstanceKind>(GetFlashSystemResource(props?.match?.params?.name, props?.match?.params?.namespace));
```
Comment (GowthamShanmugam) [inline src/components/raw-capacity-card/raw-capacity-card.tsx:45]: is both names are same? (props?.match?.params?.namespace and data?.[0]?.metadata.name)

Comment (GowthamShanmugam) [reply src/components/raw-capacity-card/raw-capacity-card.tsx:45]: I am trying to understand why watcher used here

**Review by bipuladh — APPROVED** [review]

**Pull request merged**